mantidproject/mantid
# TOPAZ: Problem with peak intensity assignment for satellite peaks

**To test:**
Run the system test downloading the following input files in the `/tmp` directory:
- [TOPAZ_39037_bank29.nxs](https://www.dropbox.com/s/5s6qlpgrci2fj0o/TOPAZ_39037_bank29.nxs?dl=0)
- [TOPAZ_39037_peaks_short.nxs](https://www.dropbox.com/s/kalr5hudyo0wvpp/TOPAZ_39037_peaks_short.nxs?dl=0)
```python
from mantid.simpleapi import *
LoadNexus(Filename='/tmp/TOPAZ_39037_bank29.nxs', OutputWorkspace='events')
LoadNexus(Filename='/tmp/TOPAZ_39037_peaks_short.nxs', OutputWorkspace='peaks_input')
IntegrateEllipsoids(InputWorkspace='events',
                    PeaksWorkspace='peaks_input',
                    OutputWorkspace='peaks_output',
                    RegionRadius=0.14,
                    SpecifySize=True,
                    PeakSize=0.07,
                    BackgroundInnerSize=0.09,
                    BackgroundOuterSize=0.11,
                    CutoffIsigI=5.0,
                    AdaptiveQBackground=True,
                    AdaptiveQMultiplier=0.001,
                    UseOnePercentBackgroundCorrection=False)
```
Verify that the two entries in column `intens` are `938` and `13936` when rounded to the nearest integer.

Refs #30737
---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
